src/metadata.json

```diff
@@ -5,6 +5,6 @@
   "settings-schema": "org.gnome.shell.extensions.battery-indicator-upower",
   "url":"https://github.com/malko/battery-indicator-upower",
   "shell-version": [
-    "42", "43"
+    "45"
```

Comment:
So in my understanding, 44 version of gnome should be added to the oldest version before publishing this one, am I right ?